### ISSUE TITLE
Is it possible to use the debugview in the video

### ISSUE THREAD

**Issue by lanmo0923** [kmammou/v-hacd#142]
![image](https://user-images.githubusercontent.com/49888704/210052292-c17f0a4a-4256-4961-ad5e-6b496743692f.png)

**Comment by jratcliff63367** [comment]
Yes, it would be reasonable to do that. It will take me a few days though and I won't be able to get to it for a while. However, I will keep this issue open and try to get to it.

**Comment by jratcliff63367** [comment]
I don't think DebugView has been integrated into this version of the tool. That is something I could do, in theory, but it would take a while. I could just upload DebugView by itself somewhere, but it would need quite a bit of documentation.
